lib/codecs/h264.js

```diff
@@ -149,6 +156,17 @@ H264Stream = function() {
   H264Stream.prototype.init.call(this);
   self = this;
 
+  /*
+   * Pushes a packet from an Elementary Stream onto the NalByteStream
```

Comment:
Same comment as above around introducing Elementary Stream here (and for comments below).

Reply:
This one I can agree can be more generic since it might actually be used in a different order